lib/components/form/submission/list.vue

```diff
@@ -181,26 +184,35 @@ export default {
     $(window).off('.form-submission-list');
   },
   methods: {
-    submissionsURL() {
-      return `/forms/${this.form.encodedId()}.svc/Submissions`;
+    chunkURL({ top, skip = 0 }) {
+      const queryString = `%24top=${top}&%24skip=${skip}`;
+      return `/forms/${this.form.encodedId()}.svc/Submissions?${queryString}`;
     },
-    processSubmissions({ data }) {
-      this.submissions = data.value != null ? data.value : [];
+    processChunk(chunkIndex, { data }) {
+      if (chunkIndex === 0)
```

Comment:
nit: if one block in if/else is braced i prefer both to be braced. nbd.

Reply:
I'm not sure how sensible my approach really is, but I usually add braces to all `if`/`else` blocks below the first block with a brace. So I'll sometimes have starting `if` and `else if` blocks without braces that are followed by blocks with braces. In general, I like avoiding braces where I can, but I also like the consistency of what you're suggesting. Do you find that consistency more readable or easier to edit?

On a separate note, I also wanted to make sure that you're looking at the latest code, since this particular line is outdated. I think you are, since you comment on newer lines below, but I just wanted to check.

Reply:
yeah i started a bit too early but did all the commits

Reply:
if it were up to me i'd leave every brace possible out. but i think for readability for others i try to stay consistent within each block. if one case needs multiple lines, they all get braces. sometimes this leads to me trying to reorganize the code so that the multiple lines aren't necessary.

Reply:
That sounds like a good approach to me. :+1: I might leave these lines as-is for now (and I suspect that there are quite a few like them throughout Frontend), but I'll use consistent braces in the future.